.github/workflows/client-pypi-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release to PyPi and upload artifact
+
+on:
```

Comment:
Don't you want to add on tag too ? To upload the tag as version.

Reply:
I do not want to do that at this moment, let's have manual control at this moment. Once we manage to push couple of releases we can switch to release on tag